contracts/sophia.md

```diff
@@ -62,11 +62,14 @@ worked out).
 Sophia does not have arbitrary mutable state, but only a limited form of
 state associated with each contract instance.
 
-- Each contract defines a state type encapsulating its mutable state.
+- Each contract defines a type `state` encapsulating its mutable state.
+- The initial state of a contract is computed by the contract's `init`
+  function. The `init` function is *pure* and returns the initial state as its
```

Comment:
Can the init function call other contracts?
Can the init function call other contracts by name?
Can the init function interrogate the chain state?

Reply:
At the moment, yes (except calling by name, which isn't implemented at all). Whether it should be allowed, not sure.